src/__tests__/update-run.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as core from '@actions/core';
```

Comment:
Let's add another tests that's doing both message + another update, let's say link